config/default/manager_image_patch.yaml

```diff
@@ -1,11 +0,0 @@
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: controller-manager
-  namespace: system
-spec:
-  template:
-    spec:
-      containers:
-      - image: gcr.io/cluster-api-ipam-provider-in-cluster/release/manager:main
```

Comment:
Have you tried setting this to the correct image instead?

Reply:
I haven't tried, but I believe it would work too. It just seemed like an extra unnecessary step considering that the Makefile uses the `IMG` environment variable.